### DISCUSSION TITLE
old topic not answered

### DISCUSSION THREAD

**Discussion by Alderaaan** [jgraph/drawio#3372] [Q&A]
Hello,
I installed the soft with docker the last week and with the last version, my docker don't at work at all.
I found an issue with the same error and already opened, but without a awsered.

How I can refresh this topic ?
https://github.com/jgraph/drawio/issues/2941

Tks.

**Comment by davidjgraph** [comment]
Thank you for your question. Please understand that draw.io is a free, open-source project developed by a small team with limited resources. We try our best to answer as many questions as possible, but unfortunately, we may not be able to answer every thread. However, we encourage you to keep posting your questions, as the community may be able to help provide answers as well. Thank you for your understanding and for being a part of the draw.io community.
